python/eos/tasks.py

```diff
@@ -517,7 +520,12 @@ def corner_plot(analysis_file:str, posterior:str, base_directory:str='./'):
             ax.set_aspect(_np.diff((xmin, xmax))[0] / _np.diff((ymin, ymax))[0])
 
     fig.tight_layout()
-    fig.savefig(os.path.join(base_directory, posterior, 'plots', 'corner-plot.pdf'))
+
+    _format = _copy.copy(format)
+    if isinstance(_format, str):
+        _format = [ _format ]
+    for f in _format:
```

Comment:
Hmm, ok... but can we do without `_copy`?:
```python
for f in np.array([format]).flatten():
    ....
```

Reply:
That would work. So does the solution with ``copy``. Is there a problem I am not seeing?